src/Histogram.ts

```diff
@@ -42,6 +47,8 @@ export default class Histogram {
     this.binSize = hinfo.binSize;
 
     // track the first and last nonzero bins with at least 1 sample
+    // TODO: If `this.min` is not 0, should `this.dataMinBin` be `0` (instead of starting at 1)
+    // since the first bin doesn't represent zero values?
```

Comment:
I guess the original intent of this was: when all the data used to always be 8bit 0-255, and there were always 255 bins, one for each intensity, then you would know that bin 0 always represented only intensity=0.   Now bin 0 could actually have nonzero intensity values in it, so looping from 1 is not necessarily doing the same thing.    

Kinda makes me wonder if we should track the number of "0" data values separately.  Which begs the question: is 0 a special value or not?

Reply:
I'd love to say zero is not a special value... I'll check where this is being used and if it's not really then I'll change it so `dataMinBin` is just `0`.

Reply:
Discussion Notes:
- In 8-bit data, the 0th bin always contains only 0 values.
- In 16+ bit data, the 0th bin can contain a mix of 0 and nonzero values.
- If we treat zero values as pixels, this would cause percentile values to behave very differently for certain types of volumes that have a lot of zero-pixels (e.g. segmentations, volumes that have "padding" in the form of zeroes)
  - How do we know if 0 is a "special" value (meaning no data) or a normal value (should be included in data calculations?)
- One solution would be to bin zero pixels separately.

Reply:
After looking at ImageJ, we decided that including the 0th bin should be the correct behavior.